packages/curve-ui-kit/src/themes/typography/mui-typography.ts

```diff
@@ -1,22 +1,45 @@
 import type { Components } from '@mui/material/styles'
+import { TypographyVariantKey } from './create-typography'
+
+const variantMapping = {
+  headingXxl: 'h1',
+  headingMBold: 'h2',
+  headingMLight: 'h3',
+  headingSBold: 'h4',
+  headingXsBold: 'h5',
+  headingXsMedium: 'h6',
+
+  bodyMRegular: 'p',
+  bodyMBold: 'p',
+  bodySRegular: 'p',
+  bodySBold: 'p',
+  bodyXsRegular: 'p',
+  bodyXsBold: 'p',
+
+  buttonXs: 'span',
+  buttonS: 'span',
+  buttonM: 'span',
+
+  tableHeaderM: 'dfn',
```

Comment:
Similarly to the issue with highlight font styling, browser apply italic styling by default to `<dfn>` elements, which is not what should happen to according to the Figma design.